### ISSUE TITLE
Make this extension Compatible with new LEETCODE Dynamic UI

### ISSUE THREAD

**Issue by shubh4122** [Edwardsoen/Leetcode-Premium-Unlocker#65]
Leetcode has recently released a new Dynamic UI feature, which helps in customising the positions, and size of panes.
But with this change, this leetcode Premium unlocker extension has stopped working.

Hence, some changes have to be made in this extension's code to adapt to new UI too.

Screenshot for clear understanding:
![image](https://github.com/Edwardsoen/Leetcode-Premium-Unlocker/assets/86017598/ff9c8609-7bd7-450c-b180-623fa3e26f34)

**Comment by adarshdhital007** [comment]
Yes , I was about to open a same issue for this. Glad, you posted it. I hope it gets fixed soon.

**Comment by adarshdhital007** [comment]
@Edwardsoen , I hope this is a critical issue to be fixed ASAP.

**Comment by Edwardsoen** [comment]
Working on it, the new dynamic UI is optional. For now disable it if you want to view companies tag data or editorial data
